config/i18n/locales/english/redirects.json

```
@@ -26013,5 +26013,10 @@
     "source": "/65e21a8b0b0c",
     "destination": "/news/zipline-build-a-smart-javascript-calculator-65e21a8b0b0c",
     "type": 302
+  },
+  {
+    "source": "/news/learn-calculus-by-coding-in-python",
```

```suggestion
    "source": "/learn-calculus-by-coding-in-python",
```

I don't think you need to add `/news` prefix to the source as none of the other redirects have it